cmd/server/main.go

```diff
@@ -248,3 +252,33 @@ func setupStorageBucket(logger log.Logger, bucketName, cloudProvider string) *fi
 	}
 	return nil
 }
+
+func setupValidationStrategies(logger log.Logger, adminServer *admin.Server) (map[validator.StrategyKey]validator.Strategy, error) {
+	strategies := map[validator.StrategyKey]validator.Strategy{}
```

Comment:
Initially I think we should have an object that takes a "validation repository" (repository that stores `validationID` and metadata) and a validationID to return a `validator.Strategy` instead of the map lookups. This would allow that factory to return errors and keep implementation details away from calling code.  Thoughts?

Reply:
I think we can do something like this. Let me first implement validation repository and then we will see how it may work.

Reply:
Sure thing. Thanks @alovak